tgui/packages/tgui/interfaces/AppearanceChanger.js

```diff
@@ -54,6 +55,7 @@ export const AppearanceChanger = (props, context) => {
     change_eye_color ||
     change_skin_tone ||
     change_skin_color ||
+    change_runechat_color ||
```

Comment:
Can we make this `true` instead of passing `TRUE` from the TGUI data send each time?

Reply:
To keep in line with the rest of the appearance changes, I'll keep it as is. It makes it more readable and digestible.
The most closely related one would be `change_eye_color` which follows the exact same principle.

![image](https://github.com/user-attachments/assets/9c574b95-cdfa-4f59-a55c-72d24551efc1)
![image](https://github.com/user-attachments/assets/c06ff50e-f44b-4da7-9b93-23ee32323780)